easybuild/easyconfigs/c/cuDNN/cuDNN-7.6.4.38-CUDA-10.0.130.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
```

Comment:
there is now a cuDNN easyblock, this can be removed

Reply:
Fixed